homeassistant/components/airthings/sensor.py

```diff
@@ -69,6 +69,7 @@
     ),
     "voc": SensorEntityDescription(
         key="voc",
+        device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
```

Comment:
This device class requires ug/m3 as unit of measurement.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes